external/myke.yml

```diff
@@ -1,6 +1,6 @@
 project: external
 desc: external services for DinoPark
 tasks:
-  run-k8s:
+  run-dev:
     cmd: |-
-      kubectl apply -f elastic/k8s/chart
\ No newline at end of file
+      kubectl apply -f elastic/k8s/chart
```

Comment:
(Optional) if you don't mind, could you move `k8s/chart/` → `k8s/tempate` and add a baisc `Chart.yaml`